src/Factories/ChoiceFactory.php

```diff
@@ -58,4 +58,19 @@ public function getConfirmationPageTemplate( Twig_Environment $twig, array $cont
 		throw new UnknownChoiceDefinition( 'Confirmation Page Template configuration failure.' );
 	}
 
+
+	public function getSkinForUserBucket(): string {
```

Comment:
I'd call this function `getSkinTemplateDirectory` and the other one `buildSkinDirectory`. Reasoning:

* the public function should clearly reflect what's returned.
* We don't have the concept of "Users" yet
* The "Bucketing" is implied by using the `ChoiceFactory` and should not show up in its method names.